dkg/disk.go

```diff
@@ -95,8 +98,43 @@ func loadDefinition(ctx context.Context, conf Config) (cluster.Definition, error
 	return def, nil
 }
 
-// writeKeystores writes the private share keystores to disk.
-func writeKeystores(datadir string, shares []share) error {
+// writeKeysToKeymanager writes validator private keyshares for the node to the provided keymanager address.
+func writeKeysToKeymanager(ctx context.Context, keymanagerURL string, shares []share) error {
+	var (
+		keystores []keystore.Keystore
+		passwords []string
+	)
+
+	for _, s := range shares {
+		password, err := testutil.RandomHex32()
```

Comment:
can't use testutil in production code, also make sure you not using math.Random, but crypto.Random.

Reply:
gotcha. The random function's using crypto/rand though it shouldn't be part of the testutil package. Will refactor this out into `eth2util` in a separate PR